docs/running-on-kubernetes.md

```diff
@@ -0,0 +1,206 @@
+---
+layout: global
+title: Running Spark on Kubernetes
+---
+
+Support for running on [Kubernetes](https://kubernetes.io/) is available in experimental status. The feature set is
+currently limited and not well-tested.
+
+## Setting Up Docker Images
+
+In order to run Spark on Kubernetes, a Docker image must be built and available on the Docker registry. Spark
+distributions include the Docker files for the driver and the executor at `dockerfiles/driver/Dockerfile` and
+`docker/executor/Dockerfile`, respectively. Use these Docker files to build the Docker images, and then tag them with
+the registry that the images should be sent to. Finally, push the images to the registry.
+
+For example, if the registry host is `registry-host` and the registry is listening on port 5000:
+
+    cd $SPARK_HOME
```

Comment:
We should also record dev-workflow docs somewhere, these aren't included in this PR just yet.